# Awesome Nascent RNA-seq analysis

A collection of tools, pipelines, and guides for performing nascent RNA sequencing analysis

# Raw data processing pipelines

- [Nascent-Flow](https://github.com/Dowell-Lab/Nascent-Flow): Nextflow Implementation of the Dowell Lab Nascent Pipeline.
- [nf-core Nascent](https://github.com/nf-core/nascent): nascent transcription processing pipeline built using Nextflow.
- [NRSA](http://bioinfo.vanderbilt.edu/NRSA/): analyze nascent transcription profiles generated by PRO-seq and GRO-seq data. NRSA quantifies nascent transcription for known genes, and detects, annotates, and quantifies active enhancers.  NRSA predicts enhancer targets based on the closest TSS and/or using enhancer-TSS associations from FANTOM5 and 4DGenome where available.
- [PEPPRO](http://peppro.databio.org/en/latest/): a pipeline for PRO-seq, GRO-seq, and ChRO-seq data that is optimized on unique features of nascent RNA to be fast and accurate. It performs variable-length UMI adapter removal, read deduplication, trimming, mapping, QC, and signal tracks (bigWig) for plus and minus strands using mappability-scaled or unscaled read counts.
- [Proseq2.0](https://github.com/Danko-Lab/proseq2.0): Preprocesses and aligns run-on sequencing (PRO/GRO/ChRO-seq) data from Single-Read or Paired-End Illumina Sequencing.

# Read quantification

- [DENR](https://github.com/CshlSiepelLab/DENR): performs annotation based transcript level quantification on nascent RNA sequencing data. Explains observed polymerase density as a weighted mixture of the underlying transcript annotations.

# Region based annotations

- [FStitch](https://github.com/Dowell-Lab/FStitch): Fast Stitch Reader (FStitch) rapidly processes read coverage files into contigs of active and inactive regions of transcription. Intended for refining annotations in nascent transcription data.
- [Tfit](https://github.com/Dowell-Lab/Tfit): Annotate sites of bidirectional transcription

# Identifying regulatory elements

- [dReg](https://github.com/Danko-Lab/dREG): Detection of Regulatory DNA Sequences using GRO-seq Data.

# Differential analyses

- [tfTarget](https://github.com/Danko-Lab/tfTarget): identify differentially regulated TF networks based on comparative conditions (e.g. Treatment -vs- Control).

# Simulating nascent sequencing

- [nascentRNASim](https://github.com/CshlSiepelLab/nascentRNASim): simulate nascent RNA-seq based on empirical data.

# Guides and walkthroughs

- [GRO-seq guide](http://homer.ucsd.edu/homer/ngs/groseq/groseq.html): GRO-seq analysis guide with HOMER.
